### DISCUSSION TITLE
Inbox shows text of minimized comments

### DISCUSSION THREAD

**Discussion by EricFromCanada** [community/community#48393] [Mobile]
### Select Topic Area

Bug

### Body

If the most recent comment of an issue or PR is minimized, its full text will still show in the Inbox list. 

![Screenshot 2023-02-23 at 09 21 00](https://user-images.githubusercontent.com/3324775/220934115-1cdb7843-4f17-491a-903c-097b9bc0ef59.png)

**Comment by candyho** [comment]
Hey, thanks for your feedback. Just want to to make sure I understood your report correctly - by "minimized comment", do you mean you have "hide" the comment in the original issue/PR, and expect that comment not to show up in the inbox view?

**Reply by EricFromCanada** [reply]
Basically, yes. I found it odd that a hidden / minimized / spam comment would have its contents shown in the inbox view.
